LOCAL_DEV.md

```diff
@@ -0,0 +1,234 @@
+# 😺 Kitty Items - Local Development 👩‍💻
+
+## 👋 Introduction
+We chose to connect Kitty Items to Flow's testnet because we wanted users to see how easy it was to take a non-trivial application to a live Flow network.
+
+Of course, some users may want to expreiment with their smart contracts before deploying them, or otherwise prototype their appliccation before publishing it.
```

Comment:
```suggestion
Of course, some users may want to experiment with their smart contracts before deploying them, or otherwise prototype their application before publishing it.
```